client/src/components/global/Topbar.tsx

```diff
@@ -98,7 +98,7 @@ export default function Topbar(): JSX.Element {
                                 Login
 										</Button>
 									): (
-										<Button onClick={() :Promise<void> => validate(user)}
+										<Button onClick={() :Promise<boolean> => validate(user)}
```

Comment:
Weird thing to do, i think we should validate the user each time a page is loaded or changed instead of onclick.